spec/providers/coreIntegration/tcpsocket.integration.src.js

```diff
@@ -131,7 +167,7 @@ module.exports = function (provider, setup) {
 
         // Wait a second before unpausing.  Data received in this second
         // will fail the expectation that paused is false in gotMessageAsync.
-        setTimeout(function() {
+        setTimeout(function () {
```

Comment:
Wait, you're saying it will give me a callback to tell me the socket is
paused and then schedule more incoming data for me to read? That sounds
crazy :fearful:

On Sat, Mar 21, 2015, 3:25 PM Benjamin M. Schwartz <notifications@github.com>
wrote:

> In spec/providers/coreIntegration/tcpsocket.integration.src.js
> <https://github.com/freedomjs/freedom/pull/241#discussion_r26897475>:
>
> > @@ -131,7 +167,7 @@ module.exports = function (provider, setup) {
> >
> >          // Wait a second before unpausing.  Data received in this second
> >          // will fail the expectation that paused is false in gotMessageAsync.
> > -        setTimeout(function() {
> > +        setTimeout(function () {
>
> Note that pause() on most platforms returns before actually guaranteeing
> that the socket is paused ... so you'd still want a timeout after calling
> pause :-(.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/freedomjs/freedom/pull/241/files#r26897475>.
>